### PR TITLE
remove references to specific services

### PR DESCRIPTION
**Pull request by openprivacy**
needed cleanup - after this, more work to do (local duplicate of #625 created from fork)

[//]: # (rtdbot-start)

URL of RTD document: https://civicactions-handbook.readthedocs.io/en/fen-sanitize/ Documentation Status

[//]: # (rtdbot-end)

### CODE REVIEW

**Review by alanna-blinn — APPROVED** [review]

File: docs/09-security/contingency-plan.md [hunk 1]
```diff
@@ -123,35 +123,33 @@ If there is a disruption in the OpsGenie service, all alerts automatically get d
...
+- IT Zoom: <https://zoom.us/>
+- Google Meet: <https://meet.google.com/>
 - Google Chat: <https://chat.google.com/>
-- IT Zoom: <https://zoom.us/j/865708118> Meeting ID: 865 708 118
-- IT Google Hangout: <https://hangouts.google.com/hangouts/_/civicactions.net/it_is>
 
 #### CPM
 
-- **Service:** <https://gnet-cpm.civicactions.net/signin/>
+- **Service:** <https://cpm.civicactions.net/>
```
Comment (grugnog) [inline docs/09-security/contingency-plan.md]: This URL doesn't exist and we don't use CPM internally or on most projects - we might just remove this section?

File: docs/09-security/contingency-plan.md [hunk 2]
```diff
@@ -96,15 +96,15 @@ CivicActions managed systems often depend on several external services. In the e
...
 disruption would only impact the team's ability to update code on the instances.
 
 #### GitLab
 
-- **Service:** <https://git.civicactions.net/dsca/>
+- **Service:** <https://git.civicactions.net/>
 - **Status:** <https://app.statuscake.com/AllStatus.php?tid=1702974>
 
-If GitLab becomes unavailable, GlobalNET will continue to operate in its current state. The disruption would only impact the team's ability to update code on the instances.
+If GitLab becomes unavailable, systems will continue to operate in its current state. The disruption would only impact the team's ability to update code on the instances.
```
Comment (grugnog) [inline docs/09-security/contingency-plan.md]: From a company contingency perspective Gitlab is pretty critical - while it doesn't affect clients services, it does affect the teams ability to work. Not sure if/how to address this here?

**Review by grugnog — CHANGES_REQUESTED** [review]

File: docs/09-security/contingency-plan.md (hunk 1 above)
Comment (openprivacy) [reply docs/09-security/contingency-plan.md]: removed CPM

File: docs/09-security/contingency-plan.md (hunk 2 above)
Comment (openprivacy) [reply docs/09-security/contingency-plan.md]: Increased note of impact.

**Review by grugnog — APPROVED** [review]

**Pull request merged**